File: Algorithm/Programmers/Python/Level 2/JadenCase 문자열 만들기.py
def solution(s):
    S=list(s)
    S[0]=S[0].upper()
    for i in range(1,len(S)):
        if S[i-1]==' ':
            S[i]=S[i].upper()
        else:
            S[i]=S[i].lower()
    return ''.join(S)

File: Algorithm/Programmers/Java/Level 2/JadenCase_문자열_만들기.java
import java.util.*;
class JadenCase_문자열_만들기 {
    public String solution(String s) {
        String[] str = s.split("");
        str[0] = str[0].toUpperCase();
        for(int idx = 1; idx < str.length; idx++) {
            if(str[idx - 1].equals(" ")) {
                str[idx] = str[idx].toUpperCase();
            } else {
                str[idx] = str[idx].toLowerCase();
            }
        }
        return String.join("", str);
    }
}